pkg/operation/botanist/controlplane.go

```diff
@@ -300,6 +293,46 @@ func (b *Botanist) HibernateControlPlane(ctx context.Context) error {
 	return nil
 }
 
+// ScaleControlPlaneDeploymentsToZero scales the kube-apiserver, kube-controller-manager and gardener-resource-manager to zero
+func (b *Botanist) ScaleControlPlaneDeploymentsToZero(ctx context.Context) error {
```

Comment:
Can we name this function something like "PrepareControlPlaneDeployments" - the actual scaling is not happening here, plus it also deletes the vpa for the APiserver.

Reply:
Sure